Bottom: 04dab5c2650cddd666620ca3972d8e8b9634e78e
Top:    3c74e5539a84303665b683bbbac358c99669b1d9
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-01-11 14:12:26 +0800

tools: fix virtio test build failure

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/tools/include/xen/xen.h b/tools/include/xen/xen.h
new file mode 100644
index 0000000..44f993c
--- /dev/null
+++ b/tools/include/xen/xen.h
@@ -0,0 +1,6 @@
+#ifndef _XEN_XEN_H
+#define _XEN_XEN_H
+
+#define xen_domain()		(0)
+
+#endif	/* _XEN_XEN_H */
